recipes/dust_extinction/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "dust_extinction" %}
+{% set version = "v0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/karllark/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c038d51d94dc65a4c6e3f473e60888f79e65e5acb07af49598012823499f34fa
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - astropy >=1.3
```

Comment:
All of these `build` requirements should be in `host` section.

Reply:
Using a different template now.  So, maybe fixed?

Reply:
I've left another comment with the change that should make the linter happy.